app/views/meetings/show.html.erb

```diff
@@ -44,7 +44,10 @@ See doc/COPYRIGHT.md for more details.
   <% end %>
   <% if authorize_for(:meetings, :destroy) %>
     <li class="toolbar-item">
-      <%= link_to({:controller => '/meetings', :action => 'destroy', :id => @meeting}, class: 'button') do %>
+      <%= link_to({controller: '/meetings', action: 'destroy', id: @meeting},
+                  class: 'button',
+                  method: :delete,
+                  confirm: l(:text_are_you_sure)) do %>
```

Comment:
I would opt for using a path helper here - altough we can wait for the Refactoring in opf/openproject#3029 before doing that.